projects/step-core/src/lib/components/edit-scheduler-task-dialog/edit-scheduler-task-dialog.component.html

```diff
@@ -0,0 +1,61 @@
+<step-modal-window [title]="isNew ? 'New task' : 'Edit task'">
+  <div step-modal-window-body>
+    <step-alert *ngIf="error" type="danger">
```

Comment:
another thing we should consider - say we have an ```enum Foo { BAR = 'bar' }``` and component ```app-something``` which has an ```@Input() foo!: Foo;```

so should we do it like ```<app-something foo="bar"></app-something``` or ```<app-something [foo]="Foo.BAR"></app-something>``` (having ```protected readonly Foo = Foo``` in the ```.ts```)